### PR TITLE
Add opentelemetry for shell scripts to the registry 

### PR DESCRIPTION
**Pull request by plengauer**
this adds project https://github.com/plengauer/opentelemetry-bash to the registry

### CODE REVIEW

File: data/registry/otel-shell.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title: OpenTelemetry for shell scripts and github actions
```
Comment (theletterf) [inline data/registry/otel-shell.yml:1]:
```suggestion
title: OpenTelemetry for shell scripts and GitHub actions
```

**Review by theletterf — COMMENTED** [review]
Pretty cool!

@svrnm I wonder if this could become part of OTel one day?

File: data/registry/otel-shell.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+title: OpenTelemetry for shell scripts and GitHub actions
```
Comment (cartermp) [inline data/registry/otel-shell.yml:1]:
```suggestion
# cSpell:ignore Philipp Lengauer
title: OpenTelemetry for shell scripts and GitHub actions
```

**Review by cartermp — APPROVED** [review]
This should get things to go green.

**Pull request merged**